app/components/account/MetaplexNFTAttributesCard.tsx

```diff
@@ -14,7 +14,7 @@ export function MetaplexNFTAttributesCard({ nftData }: { nftData: NFTData }) {
 
     async function fetchMetadataAttributes() {
         try {
-            const response = await fetch(nftData.metadata.data.uri);
+            const response = await fetch(nftData.metadata.uri);
```

Comment:
Can you rebase & use `getProxiedUri(nftData.metadata.uri)` here please?

Reply:
Oh no I'm back in lockfile hell 😭 . Let me try and resolve some of the dep changes.